cli/internal/packager/create.go

```diff
@@ -52,7 +53,9 @@ func Create() {
 
 	if len(seedImages) > 0 {
 		// Load seed images into their own happy little tarball for ease of import on init
-		images.PullAll(seedImages, tempPath.seedImages)
+		pulledImages := images.PullAll(seedImages, tempPath.seedImages)
+		_ = utils.CreateDirectory(tempPath.sboms, 0700)
+		sbom.CatalogImages(pulledImages, tempPath.sboms)
```

Comment:
@jeff-mccoy This is writing the sboms to the temp directory which I believe gets blown away when the package creation is finished (unless there is a fatal error). Is there a persistent but intuitive place you think these should be written to instead?

Reply:
From my testing the sboms directory is getting packaged into the .zst

Reply:
Oops you're totally correct, I forgot we archive the temp directory into the package itself.